packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -830,6 +840,61 @@ class _SelectableRegionState extends State<SelectableRegion> with TextSelectionD
     await Clipboard.setData(ClipboardData(text: data.plainText));
   }
 
+  bool? _adjustingSelectionExtent;
+  bool _determineIsAdjustingSelectionExtend(bool forward) {
```

Comment:
"Extend" => "Extent"?

Reply:
I renamed it to End which is more accurate